package.json

```diff
@@ -12,44 +12,27 @@
   "scripts": {
     "build": "ng-packagr -p package.json"
   },
-  "author": "Wilson Hobbs",
-  "license": "MIT",
   "peerDependencies": {
-    "@angular/core": "8.2.14",
-    "@ionic/angular": ">=4",
-    "@types/hammerjs": "^2.0.36",
-    "hammerjs": "^2.0.8"
+    "@angular/core": "12.x",
```

Comment:
does this syntax actually work? I've seen `^12.0.0` and `>=12.0.0` work, but I have never actually seen this syntax

Reply:
yep! It was new to me too when I was researching peer deps.

source: https://nodejs.org/es/blog/npm/peer-dependencies/